sustainability.html

```diff
@@ -156,7 +156,7 @@ <h3>One global (green) team</h3>
                         </div>
                     </section>
                     <!-- End Service Section -->
-                    <hr />
+                    <!-- <hr /> -->
```

Comment:
Remove commented code

Reply:
only commented the line separator it is part of the section commented below this line, hence it is kept.